build.sh

```diff
@@ -36,35 +42,47 @@ bash ./start-rc.sh
 sleep 10
 
 # Run tests (JUnit plugin) with race detection
-for pkg in $(go list $CLIENT_IMPORT_PATH/...);
-do
-    if [[ $pkg != *"vendor"* ]]; then
-      echo "testing with race detection on ... $pkg"
-      go test -race -v  $pkg
-    fi
-done
+#for pkg in $(go list $CLIENT_IMPORT_PATH/...);
```

Comment:
Please remove commented out code block

Reply:
I will clean up the script after we agree on the essentials